zzz_generated/app-common/app-common/zap-generated/cluster-enums.h

```diff
@@ -2460,7 +2460,7 @@ enum class SensitivityEnum : uint8_t
 enum class Feature : uint32_t
 {
     kSmokeAlarm = 0x1,
-    kCoAlarm    = 0x2,
+    kCOAlarm    = 0x2,
```

Comment:
This is the source-compat-breaking change.  A change for the better mind you, but still....

Reply:
Yeah, I created the draft to run the CI checks while I performed some tests, first thing I noticed is that this change is breaking the compilation of example apps (e.g. all_clusters).